tools/src/ExpoCLI.ts

```diff
@@ -26,3 +28,18 @@ export async function runExpoCliAsync(
     },
   });
 }
+
+export async function extractTarballAsync(tarFilePath: string, props: ExtractProps): Promise<void> {
+  return await extractLocalNpmTarballAsync(tarFilePath, props);
+}
```

Comment:
What's the difference between these two?

Reply:
There is no difference.